pytket/docs/index.rst

```diff
@@ -128,6 +128,7 @@ Licensed under the `Apache 2 License <http://www.apache.org/licenses/LICENSE-2.0
     
     Manual <https://cqcl.github.io/pytket/manual/index.html>
     Extensions <https://cqcl.github.io/pytket-extensions/api/index.html>
+    Example notebooks <https://github.com/CQCL/pytket/tree/main/examples>
```

Comment:
```suggestion
    Example notebooks <https://github.com/CQCL/pytket/tree/main/examples#pytket-examples>
```

Reply:
How about this?

Reply:
I tried out the suggested change and this seems to redirect to the README of the examples repo. Is this what you intended? If so this is fine with me.

Is there another file I should change as well?